pkg/strategy/dca2/strategy.go

```diff
@@ -66,6 +66,9 @@ type Strategy struct {
 	// UseCancelAllOrdersApiWhenClose uses a different API to cancel all the orders on the market when closing a grid
 	UseCancelAllOrdersApiWhenClose bool `json:"useCancelAllOrdersApiWhenClose"`
 
+	// dev mode
+	DevMode DevMode `json:"devMode"`
```

Comment:
would be better to use `DevMode *DevMode`

Reply:
I think if we use pointer here, we can remove Enabled field.